diff --git a/node_modules/jsqr/dist/jsQR.js b/node_modules/jsqr/dist/jsQR.js
index 51571d8..8e17812 100644
--- a/node_modules/jsqr/dist/jsQR.js
+++ b/node_modules/jsqr/dist/jsQR.js
@@ -524,4 +524,4 @@
     return jsQR;
 }));
 //# sourceMappingURL=jsQR.js.map
-//# sourceURL=webpack-internal:///./node_modules/jsqr/dist/jsQR.js
\ No newline at end of file
+//# sourceURL=webpack-internal:///./node_modules/jsqr/dist/jsQR.js
